src/test/java/seedu/recipe/logic/commands/FindCommandTest.java

```diff
@@ -84,8 +87,10 @@ public void execute_namePropertyZeroKeywords_noRecipeFound() {
     public void execute_namePropertyMultipleKeywords_multipleRecipesFound() {
```

Comment:
`execute_namePropertyMultipleKeywords_twoRecipesFound`, since we have multi below too it's hard to disambiguate

Reply:
this is standardised among both name and tag